aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -9,6 +9,27 @@
 
 namespace at { namespace native {
 
+template<typename T>
+T __host__ __device__ abs_impl(T v) {
+  return std::abs(v);
```

Comment:
@VitalyFedyunin , std::abs can not be overloaded by ROCM, do you have any suggestion? using ::abs?

Reply:
Yes. `::abs` is also overloaded in CUDA/ROCm.